### PR TITLE
[ETP-810] multiple ticket forms on same page

### PR DESCRIPTION
**Pull request by ScottGarrison**
### 🎫 Ticket

[[ETP-810](https://theeventscalendar.atlassian.net/browse/ETP-810)]

### 🗒️ Description

This is the first half of PR https://github.com/the-events-calendar/event-tickets-plus/pull/1350. They should be tested together.

Fixes multiple of the same ticket form being on the same page being out of sync.

Solution - Updated some JS so that when you update the amount of tickets on one form, it will find the other form and update it in the same spot.

This ticket has a second part to it in ET+ where it fixes the Attendee forms being output in a modal in the same situation of multiple of the same forms.

### 🎥 Artifacts 
https://www.loom.com/share/6fcc78aa67f745579ccfb1e59bff90e7

### ✔️ Checklist
- [ ✔️] I've included a changelog entry in the readme.txt file. 
- [✔️ ] My code is tested. 
- [ ✔️] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

### CODE REVIEW

File: src/resources/js/v2/tickets-block.js [hunk 1]
```diff
@@ -279,6 +279,28 @@ tribe.tickets.block = {
...
+	 * @since TBD
+	 *
+	 * @param {jQuery} $input The ticket input being changed.
+	 * @param {number} newValue The value being updated to.
+	 *
+	 * @return {void}
+	 */
+	obj.formMatch = function ( $input, newValue ) {
+		var formID = $input.closest( obj.selectors.item ).attr( 'data-ticket-id' );
+		var $formTicket = $( `[ data-ticket-id="${ formID }" ]` );
```
Comment (codingmusician) [inline src/resources/js/v2/tickets-block.js:294]:
```suggestion
		var $formTicket = $( `[ data-ticket-id="${formID}" ]` );
```

Comment (ScottGarrison) [reply src/resources/js/v2/tickets-block.js:294]: since it seems our guide has spaces around brackets, is there any reason to remove it here?

Comment (codingmusician) [reply src/resources/js/v2/tickets-block.js:294]: No reason. Thanks for clarifying.

File: src/resources/js/v2/tickets-block.js [hunk 2]
```diff
@@ -322,6 +344,9 @@ tribe.tickets.block = {
 		} else {
 			$input.val( newValue );
 		}
+
+		// If there's two of the same forms on a page, match the value of attendees on both on change
```
Comment (codingmusician) [inline src/resources/js/v2/tickets-block.js:348]:
```suggestion
		// If there's two of the same forms on a page, match the value of attendees on both on change.
```

File: src/resources/js/v2/tickets-block.js [hunk 3]
```diff
@@ -349,6 +374,9 @@ tribe.tickets.block = {
 		} else {
 			$input[ 0 ].value = decrease;
 		}
+
+		// If there's two of the same forms on a page, match the value of attendees on both on change
```
Comment (codingmusician) [inline src/resources/js/v2/tickets-block.js:378]:
```suggestion
		// If there's two of the same forms on a page, match the value of attendees on both on change.
```

**Review by bordoni — DISMISSED** [review]

**Comment by bordoni** [comment]
Need to resolve conflicts and make sure tests are passing.

**Pull request merged**